source/locale/zh_CN/LC_MESSAGES/tutorial/10_key_features/003_efficient_optimization_algorithms.po

```diff
@@ -0,0 +1,333 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) 2018, Optuna Contributors.
+# This file is distributed under the same license as the Optuna package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2021.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: Optuna 2.4.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2021-01-14 18:04-0500\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Babel 2.8.0\n"
+
+#: ../../source/tutorial/10_key_features/003_efficient_optimization_algorithms.rst:6
+msgid ""
+"Click :ref:`here "
+"<sphx_glr_download_tutorial_10_key_features_003_efficient_optimization_algorithms.py>`"
+"     to download the full example code"
+msgstr ""
+
+#: ../../source/tutorial/10_key_features/003_efficient_optimization_algorithms.rst:15
+msgid "Efficient Optimization Algorithms"
+msgstr "高效的优化算法"
+
+#: ../../source/tutorial/10_key_features/003_efficient_optimization_algorithms.rst:17
+msgid ""
+"Optuna enables efficient hyperparameter optimization by adopting state-"
+"of-the-art algorithms for sampling hyperparameters and pruning "
+"efficiently unpromising trials."
+msgstr "通过采用最先进的超参数采样算法和对无望 trial 的剪枝， Optuna使得高效的超参数优化成为可能。"
+
+#: ../../source/tutorial/10_key_features/003_efficient_optimization_algorithms.rst:22
+msgid "Sampling Algorithms"
+msgstr "采样算法"
+
+#: ../../source/tutorial/10_key_features/003_efficient_optimization_algorithms.rst:24
+msgid ""
+"Samplers basically continually narrow down the search space using the "
+"records of suggested parameter values and evaluated objective values, "
+"leading to an optimal search space which giving off parameters leading to"
+" better objective values. More detailed explanation of how samplers "
+"suggest parameters is in :class:`optuna.samplers.BaseSampler`."
+msgstr ""
+"利用 suggested 参数值和评估的目标值的记录，"
+"采样器基本上不断缩小搜索空间，"
+"直到找到一个最佳的搜索空间，其产生的参数会带来"
+" 更好的目标函数值。关于采样器如何 suggest 参数的更详细的解释见 :class:`optuna.samplers.BaseSampler`."
+
+
+
+#: ../../source/tutorial/10_key_features/003_efficient_optimization_algorithms.rst:28
+msgid "Optuna provides the following sampling algorithms:"
+msgstr "Optuna 提供了下列采样算法："
+
+#: ../../source/tutorial/10_key_features/003_efficient_optimization_algorithms.rst:30
+msgid ""
+"Tree-structured Parzen Estimator algorithm implemented in "
+":class:`optuna.samplers.TPESampler`"
+msgstr ":class:`optuna.samplers.TPESampler` 实现的树形的 Parzen Estimator 算法"
```

Comment:
`Tree-structured Parzen Estimator` is a proper noun, and we might not need to translate it.

Reply:
Agreed